### ISSUE TITLE
IE is broken- Fragment shader failed to compile

### ISSUE THREAD

**Issue by hpinkos** [AnalyticalGraphicsInc/cesium#6038]
IE is broken in master right now  @lilleyse @bagnell 

```
RuntimeError: Fragment shader failed to compile.  Compile log: Shader compilation errors
(416, 56): syntax error, unexpected IDENTIFIER, expecting RIGHT_PAREN

Error
   at RuntimeError (http://localhost:8080/Source/Core/RuntimeError.js:42:13)
   at createAndLinkProgram (http://localhost:8080/Source/Renderer/ShaderProgram.js:230:17)
   at initialize (http://localhost:8080/Source/Renderer/ShaderProgram.js:451:9)
   at ShaderProgram.prototype._bind (http://localhost:8080/Source/Renderer/ShaderProgram.js:468:9)
   at beginDraw (http://localhost:8080/Source/Renderer/Context.js:937:9)
   at Context.prototype.draw (http://localhost:8080/Source/Renderer/Context.js:1000:9)
   at DrawCommand.prototype.execute (http://localhost:8080/Source/Renderer/DrawCommand.js:502:9)
   at executeCommand (http://localhost:8080/Source/Scene/Scene.js:1722:13)
   at executeCommands (http://localhost:8080/Source/Scene/Scene.js:1977:17)
   at executeCommandsInViewport (http://localhost:8080/Source/Scene/Scene.js:2472:9)
```

**Comment by emackey** [comment]
`git bisect` says 33eb86a61aec28f0.  Sorry @ggetz!

**Comment by ggetz** [comment]
I'm pretty sure the culprit here is the array parameters for the `discardIfClipped` functions (https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Shaders/Builtin/Functions/discardIfClipped.glsl#L12), but I'm not sure what is the correct way to format this so it compiles in IE?

**Comment by pjcozzi** [comment]
I was surprised when I first saw this code.  Did you try
```
vec4 clippingPlanes[czm_maxClippingPlanes]
```

**Comment by ggetz** [comment]
Yes, it compiles, but breaks functionality (again only in IE), so I have to see what the problem is there.

**Comment by pjcozzi** [comment]
Could also be an IE bug.  If so, let's report it, but also find a workaround.

**Comment by ggetz** [comment]
I opened #6048 to track the functionality issue, so we can resolve the Fragment shader failed to compile error first.

**Comment by hpinkos** [comment]
Fixed in #6045
